src/spikeinterface/extractors/tests/common_tests.py

```diff
@@ -52,8 +52,11 @@ def test_open(self):
                 num_samples = rec.get_num_samples(segment_index=segment_index)
 
                 full_traces = rec.get_traces(segment_index=segment_index)
-                assert full_traces.shape == (num_samples, num_chans)
-                assert full_traces.dtype == dtype
+                assert full_traces.shape == (
```

Comment:
Something I like to do is to write expected_shape = (num_samples, num_channels) and then do the assertion so if I Jump in the debugger I have all the variables already loaded. But I think this is OK.

Reply:
Yeah I didn't change anything here. I was just getting an error and couldn't see what it was so I rage added this :)

Reply:
Yes, I realize, just sharing a pattern. I was going to suggest adding string to the message but this is the testing suite so I think this is fine and your addition is an improvement.